### PR TITLE
switch to safe buffer

### LINKED ISSUE #15: Moving away from new Buffer(...)
`new Buffer(...)` is deprecated and unsafe. Would it be possible to replace its usage with `Buffer.from(...)`?

### PR DESCRIPTION
**Pull request by calvinmetcalf**
fixes #15 and  nodejs/node#21351

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
also thought I'd add standard while I was in here

File: mgf.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
-var createHash = require('create-hash');
+var createHash = require('create-hash')
+var Buffer = require('safe-buffer').Buffer
+
 module.exports = function (seed, len) {
-  var t = new Buffer('');
-  var  i = 0, c;
+  var t = Buffer.alloc(0)
+  var i = 0; var c
```
Comment (dcousens) [inline mgf.js]: this is odd

Comment (calvinmetcalf) [reply mgf.js]: I think it's an artifact of standard maybe, I can fix it

**Comment by dcousens** [comment]
LGTM

**Pull request merged**